owncloudComLibrary/src/main/java/com/owncloud/android/lib/resources/files/RemoteFile.java

```diff
@@ -85,6 +85,7 @@ public RemoteFile createFromParcel(Parcel source) {
     private BigDecimal mQuotaUsedBytes;
     private BigDecimal mQuotaAvailableBytes;
     private String mPrivateLink;
+    private String mOwner;
```

Comment:
I guess this is needed because the account name and user name is not always the same, right?

Reply:
Not at all, i just want to have it earlier. Previously it was included in FileDataStorageManager, and i think it is easier to handle it here, so localdatasource won't need to ask the current account to add this attribute.